extension/js/tree.js

```diff
@@ -305,7 +299,9 @@ function populateDataIntoCiteBox(citeBox, data) {
 
 	sourceNode.innerText = `"` + data.source + `"`;
 	linkNode.innerText = data.link;
-	linkNode.href = data.link;
+	
+	console.log(`changing ${data.link} into ${data.link.split('#')[0]+'#:~:text='+escape(data.source.toLowerCase().trim())}`)
+	linkNode.href = data.link.split('#')[0]+'#:~:text='+escape(data.source.toLowerCase().trim());
```

Comment:
following the comment above, putting the function in a helpers file would let it be used here as well